src/main/java/selenuimTestWeb_button/googleTest.java

```diff
@@ -0,0 +1,27 @@
+package selenuimTestWeb_button;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class googleTest {
+ 
+	public static void main(String[] args) throws InterruptedException {
+		// TODO Auto-generated method stub
+WebDriver driver = new ChromeDriver();
+driver.get("https://www.google.com");
+driver.manage().window().maximize();
+Thread.sleep(1000);
```

Comment:
Please try to avoid time delay , as it delay the execution of the tests , we can notice that when we have several tests files